### PR TITLE
[Merged by Bors] - create a GRPC stress test

### PR DESCRIPTION
**Pull request by AmitShaul**
## Motivation
Create a stress test for our grpc

### CODE REVIEW

File: tests/stress/grpc_stress/test_stress_grpc.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    queue = mp.Queue()
+    for x in range(new_accounts):
+        # get acc pub
+        acc_pub = list(accountant.accounts.keys())[x]
+        # select one of the api calls to run randomly
+        func = random.choice(api_funcs_lst)
+        args = (acc_pub, )
+        if func == actions.transfer:
+            dst = random.choice(list(accountant.accounts.keys()))
+            amount = 1
```
Comment (antonlerner) [inline tests/stress/grpc_stress/test_stress_grpc.py]: why only 1? can we have a random value for this?

Comment (AmitShaul) [reply tests/stress/grpc_stress/test_stress_grpc.py]: sure

Comment (AmitShaul) [reply tests/stress/grpc_stress/test_stress_grpc.py]: a665e951bdb7cef193f4e3378ae89807a2489f1d

**Review by antonlerner — APPROVED** [review]

**Comment by AmitShaul** [comment]
bors merge

**Comment by AmitShaul** [comment]
bors merge

**Comment by AmitShaul** [comment]
bors merge

**Pull request closed without merging**